# Partículas — boletín II

### Observables: sección eficaz, luminosidad, vida media, invariante $s$, fracción y anchura

Jose A. Hernando

*Departamento de Física de Partículas. Universidade de Santiago de Compostela*

Curso 2026/27

In [1]:
# Configuración e imports
%matplotlib inline

import numpy as np
from scipy.stats import expon

# Constantes y masas (GeV, s, cm)
hbar  = 6.582e-25      # GeV s
c     = 3.0e10         # cm/s
N_A   = 6.022e23       # 1/mol
m_p   = 0.93827        # GeV
m_pi0 = 0.13498        # GeV
m_tau = 1.7769         # GeV
m_Z   = 91.19          # GeV
nb, pb = 1e-33, 1e-36  # cm^2

## Objetivos

Los ejercicios siguen el orden de la sección *Observables* del capítulo de introducción:

* **Sección eficaz y luminosidad** (1): la luminosidad de un blanco fijo.
* **Vida media** (2): cuántos pares se producen y cuánto vuelan.
* **El invariante $s$** (3–8): masa invariante, un pico sobre el fondo (con una gráfica y con datos
  reales), colisionador frente a blanco fijo y energías umbral.
* **Fracción de desintegración** (9) y **anchura** (10).

**Notación.** Unidades naturales, $\hbar = c = 1$, salvo que se diga lo contrario. $e$, $\mu$, $\tau$ son los
leptones con carga negativa y $e^+$, $\mu^+$, $\tau^+$ sus antipartículas; el signo $+$ separa todas las
partículas. LAB es el sistema del laboratorio y CM el del centro de masas; salvo que se diga lo contrario,
los datos están en el LAB.

## Sección eficaz y luminosidad

<div class="admonition tip">
<p class="admonition-title"><b>[>] Ejercicio 1 — Luminosidad de un blanco fijo</b></p>

Calcula la luminosidad equivalente de un haz de protones de intensidad $10^{13}$ $\mathrm{s}^{-1}$ que incide sobre
un tanque de 10 cm de longitud de hidrógeno líquido, de densidad $\rho = 60$ kg/$\mathrm{m}^3$. Compárala con la del
LHC, $10^{34}$ $\mathrm{cm}^{-2}\,\mathrm{s}^{-1}$.

</div>

In [2]:
I    = 1e13                      # protones/s
rho  = 0.060                     # g/cm^3
l    = 10                        # cm
n    = rho * N_A / 1.008         # protones (núcleos de H) por cm^3
lum  = I * n * l
print('n = {:.1e} cm^-3,  L = {:.1e} cm^-2 s^-1,  L / L_LHC = {:.0f}'.format(n, lum, lum / 1e34))

n = 3.6e+22 cm^-3,  L = 3.6e+36 cm^-2 s^-1,  L / L_LHC = 358


<details>
<summary><b>Solución</b></summary>

En blanco fijo, $\nu = \phi\,\sigma\,N_b = (I/S)\,\sigma\,(n S \ell) = \sigma\,I\,n\,\ell$; comparando con
$\nu = \mathcal{L}\sigma$, $\mathcal{L} = I\,n\,\ell$ (no depende de la sección del haz).
$n = \rho N_A/A = 3.6\times10^{22}$ $\mathrm{cm}^{-3}$, luego $\mathcal{L} = 3.6\times10^{36}$ $\mathrm{cm}^{-2}\,\mathrm{s}^{-1}$,
unas 400 veces la del LHC. (Aproximación de blanco delgado: vale si $\ell \ll \lambda$; aquí
$\lambda \sim 10$ m para $\sigma \sim 30$ mb.)

</details>

## Vida media

<div class="admonition tip">
<p class="admonition-title"><b>[>] Ejercicio 2 — Pares $\tau + \tau^+$ en el pico del $Z$</b></p>

La sección eficaz $e + e^+ \to \tau + \tau^+$ es 1.7 nb a $\sqrt s = 91$ GeV.

1. Si la luminosidad es $10^{32}$ $\mathrm{cm}^{-2}\,\mathrm{s}^{-1}$, ¿cuál es la razón de producción de pares $\tau + \tau^+$?
2. Calcula la energía, el momento y la distancia media recorrida por los $\tau$ ($\tau_\tau = 0.29$ ps).
3. ¿Es posible que un $\tau$ concreto recorra 4 mm? ¿Con qué probabilidad?
4. En el plano transverso, ¿qué relación guardan los momentos del $\tau$ y del $\tau^+$?

</div>

In [3]:
sigma, lum = 1.7 * nb, 1e32
E   = 91 / 2
p   = np.sqrt(E**2 - m_tau**2)
d   = p / m_tau * c * 0.29e-12           # gamma beta c tau, en cm
print('nu = {:.2f} /s'.format(sigma * lum))
print('E = {:.1f} GeV, p = {:.2f} GeV, gamma beta = {:.1f}, <d> = {:.2f} mm'.format(E, p, p / m_tau, 10 * d))
print('P(d > 4 mm) = {:.2f}'.format(expon.sf(0.4, scale=d)))

nu = 0.17 /s
E = 45.5 GeV, p = 45.47 GeV, gamma beta = 25.6, <d> = 2.23 mm
P(d > 4 mm) = 0.17


<details>
<summary><b>Solución</b></summary>

1. $\nu = \sigma\mathcal{L} = 1.7\times10^{-33}\times10^{32} = 0.17$ $\mathrm{s}^{-1}$.
2. $E = \sqrt s/2 = 45.5$ GeV, $p = 45.47$ GeV, $\gamma\beta = p/m = 25.6$;
   $\langle d\rangle = \gamma\beta c\tau = 25.6\times87\,\mu\mathrm{m}$ $= 2.2$ mm.
3. Sí: la distancia sigue una exponencial de media 2.2 mm; $P(d > 4\ \mathrm{mm}) = e^{-4/2.2} = 0.16$.
4. Opuestos e iguales en módulo (el CM coincide con el laboratorio).

</details>

## El invariante $s$

<div class="admonition tip">
<p class="admonition-title"><b>[>] Ejercicio 3 — Masa invariante de tres protones</b></p>

Tres protones de $p = 3$ GeV se emiten en un plano formando $120^\circ$ entre sí. ¿Cuál es su masa invariante?

</div>

<details>
<summary><b>Solución</b></summary>

Los tres momentos suman cero, así que $M = \sum E_i = 3\sqrt{9 + 0.938^2} = 9.43$ GeV.

</details>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Ejercicio 4 — Un pico sobre el fondo</b></p>

La figura muestra los sucesos con **cuatro leptones** (dos pares $e + e^+$ o $\mu + \mu^+$) que registró
ATLAS en colisiones $p + p$ a $\sqrt s = 13.6$ TeV (29 $\mathrm{fb}^{-1}$), en función de la masa invariante de los
cuatro, $m_{4l}$. Los puntos son los datos; los histogramas, lo que se espera de los procesos conocidos que
también dan cuatro leptones (el *fondo*).

| |
| :--: |
| <img src="../imgs/higgs_4l_sin_senal.png" width = 420 align = "center"> |
| $m_{4l}$ en ATLAS, con el fondo esperado [ATLAS, $\sqrt s = 13.6$ TeV; figura retocada]. |

1. ¿Hay algo en los datos que el fondo no explique? ¿Dónde? Estima cuántos sucesos de más hay y la masa de
   la partícula que los produce.
2. En uno de esos sucesos se miden, en el LAB, los momentos (en GeV)
   $e: (54, 0, 0)$, $e^+: (-38, 0, 0)$, $\mu: (0, 0, 19)$, $\mu^+: (-8, 15, 0)$.
   Desprecia la masa de los leptones. Calcula la masa invariante de los cuatro. ¿Es compatible con el pico?
   ¿Por qué se usa la masa invariante y no la energía total de los cuatro leptones?
3. Calcula la masa invariante del par $e + e^+$ y la del par $\mu + \mu^+$. ¿Qué partícula se desintegra en
   cada par? ¿Qué partícula es la madre?
4. ¿Por qué uno de los dos bosones intermedios no puede tener su masa?

</div>

In [4]:
def masa(*ps):
    # masa invariante de partículas sin masa, dados sus trimomentos
    E = sum(np.linalg.norm(p) for p in ps)
    P = np.sum(ps, axis=0)
    return np.sqrt(E**2 - P @ P)

e, ep, mu, mup = map(np.array, [(54, 0, 0), (-38, 0, 0), (0, 0, 19), (-8, 15, 0)])
print('m4l = {:.1f} GeV,  m_ee = {:.1f} GeV,  m_mumu = {:.1f} GeV'.format(
      masa(e, ep, mu, mup), masa(e, ep), masa(mu, mup)))

m4l = 125.4 GeV,  m_ee = 90.6 GeV,  m_mumu = 25.4 GeV


<details>
<summary><b>Solución</b></summary>

| |
| :--: |
| <img src="../imgs/higgs_4l_atlas.png" width = 420 align = "center"> |
| La figura original, con la señal del Higgs (azul) y la incertidumbre del fondo (rayado). |

1. Entre 120 y 130 GeV hay $8 + 11 + 10 + 10 + 7 = 46$ sucesos donde el fondo espera $\simeq 17$: unos
   **30 sucesos de más**, agrupados en un pico en $m_{4l} \simeq 125$ GeV. Fuera del pico los datos siguen al
   fondo (con fluctuaciones de $\pm\sqrt N$). Una partícula de 125 GeV que se desintegra en cuatro leptones:
   el **bosón de Higgs**. La anchura del pico, de unos pocos GeV, es la resolución del detector; la del Higgs
   es de 4 MeV.
2. $E = 54 + 38 + 19 + 17 = 128$ GeV y $\vec P = (8, 15, 19)$ GeV, $|\vec P|^2 = 650$ $\mathrm{GeV}^2$:
   $m_{4l} = \sqrt{16384 - 650} = 125.4$ GeV, en el pico. La energía total depende del sistema de referencia
   (aquí el sistema de los cuatro leptones se mueve con $\beta = 25.5/128 = 0.20$), y de un suceso a otro
   cambia con el movimiento del Higgs; la masa invariante es la misma en todos los sistemas y es la masa de la
   madre.
3. $m_{ee} = \sqrt{92^2 - 16^2} = 90.6$ GeV: un $Z$. $m_{\mu\mu} = \sqrt{36^2 - 650} = 25.4$ GeV: otro $Z$,
   pero lejos de su masa. La cadena es $H \to Z + Z^* \to e + e^+ + \mu + \mu^+$.
4. Dos $Z$ con su masa necesitan $2m_Z = 182$ GeV $> m_H = 125$ GeV: no hay energía. Uno sale con su masa y
   el otro, **virtual** ($Z^*$), con la que sobra: $m_{Z^*} \le m_H - m_Z \simeq 34$ GeV. Por eso el segundo
   par tiene siempre una masa invariante pequeña.

</details>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Ejercicio 5 — El Higgs en los datos abiertos de CMS (con el ordenador)</b></p>

Rehaz la figura del ejercicio anterior con datos reales: los 278 sucesos con cuatro leptones de CMS de 2011 y
2012 ($\sqrt s = 7$ y 8 TeV, 14 $\mathrm{fb}^{-1}$), publicados en el CERN Open Data Portal. El fichero
`datos/higgs_4l_cms.csv` da los cuadrimomentos de los cuatro leptones de cada suceso, y
`datos/higgs_4l_cms_esperados.csv`, los sucesos esperados de cada fondo y de un Higgs de 125 GeV, en
intervalos de 3 GeV de $m_{4l}$.

1. Calcula $m_{4l}$ de cada suceso a partir de los cuadrimomentos.
2. Empareja los leptones en dos pares de carga opuesta y mismo sabor; llama $Z_1$ al más cercano a $m_Z$.
   Histograma $m_{Z_1}$ y $m_{Z_2}$ para los sucesos con $121 < m_{4l} < 130$ GeV. ¿Qué relación hay entre
   $m_{Z_1}$, $m_{Z_2}$ y $m_{4l}$?
3. Dibuja el espectro de $m_{4l}$ entre 70 y 181 GeV sobre los fondos apilados. ¿De qué es el pico en 91 GeV?
   ¿Dónde hay un exceso?
4. Cuenta los sucesos observados y el fondo esperado en $121 < m_{4l} < 130$ GeV. ¿Qué probabilidad hay de
   que el fondo solo dé tantos o más? Calcúlala con la distribución de Poisson y simulando experimentos.
   ¿Es un descubrimiento?

**Este ejercicio se hace en su propio cuaderno guiado:
[boletin_II_2627_higgs_4l.ipynb](boletin_II_2627_higgs_4l.ipynb)**, con un taller por apartado y sus soluciones.

</div>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Ejercicio 6 — El LHC en blanco fijo</b></p>

En el LHC colisionan protones con $\sqrt s = 14$ TeV. ¿Qué energía tiene cada protón? Si la colisión fuese en
blanco fijo, ¿qué energía necesitaría el protón incidente?

</div>

In [5]:
s = 14e3**2                               # GeV^2
E_fijo = (s - 2 * m_p**2) / (2 * m_p)
print('colisionador: E = {:.0f} GeV;  blanco fijo: E = {:.2e} GeV = {:.1e} eV'.format(
      np.sqrt(s) / 2, E_fijo, E_fijo * 1e9))

colisionador: E = 7000 GeV;  blanco fijo: E = 1.04e+08 GeV = 1.0e+17 eV


<details>
<summary><b>Solución</b></summary>

En el colisionador (CM), $s = (2E)^2$: $E = 7$ TeV. En blanco fijo, $s = (E + m_p)^2 - p^2 = 2m_p^2 + 2m_pE$:

$$
E = \frac{s - 2m_p^2}{2m_p} \simeq \frac{(14\times10^{3})^2}{2\times0.938}\ \mathrm{GeV} = 1.04\times10^{8}\ \mathrm{GeV}
= 1.04\times10^{5}\ \mathrm{TeV} \simeq 10^{17}\ \mathrm{eV},
$$

energía que solo alcanzan los rayos cósmicos.

</details>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Ejercicio 7 — Producción de antiprotones</b></p>

Para producir antiprotones se hace incidir un haz de protones sobre un blanco de hidrógeno mediante
$p + p \to p + p + p + \bar p$.

1. Comprueba que la reacción está permitida.
2. Determina la energía mínima en el CM.
3. Determina la energía y el momento umbral del protón incidente en el laboratorio.

</div>

In [6]:
s_min = (4 * m_p)**2
E = (s_min - 2 * m_p**2) / (2 * m_p)
print('sqrt(s)_min = {:.2f} GeV;  E = {:.2f} GeV, p = {:.2f} GeV, T = {:.2f} GeV'.format(
      np.sqrt(s_min), E, np.sqrt(E**2 - m_p**2), E - m_p))

sqrt(s)_min = 3.75 GeV;  E = 6.57 GeV, p = 6.50 GeV, T = 5.63 GeV


<details>
<summary><b>Solución</b></summary>

1. Conserva la carga ($2 \to 2$) y el número bariónico ($2 \to 3 - 1 = 2$): el $\bar p$ se crea junto a un protón.
2. $\sqrt{s}_{\min} = 4m_p = 3.75$ GeV.
3. $2m_p^2 + 2m_pE = 16m_p^2 \Rightarrow E = 7m_p = 6.57$ GeV; $p = \sqrt{48}\,m_p = 6.50$ GeV; $T = 6m_p = 5.63$ GeV.

</details>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Ejercicio 8 — Fotoproducción de piones y el corte GZK</b></p>

Considera $\gamma + p \to p + \pi^0$.

1. ¿Cuál es la energía mínima de los fotones si inciden sobre protones en reposo?
2. ¿Qué energía mínima deben tener los protones ultrarrelativistas de los rayos cósmicos para que la reacción
   tenga lugar con los fotones de 1 meV del fondo cósmico de microondas?
3. Justo por encima del umbral la sección eficaz es $\sigma = 0.6$ mb y la densidad de fotones del fondo es
   $\rho_\gamma = 4.1\times10^{8}$ $\mathrm{m}^{-3}$. Calcula la longitud de interacción del protón. ¿Es grande o
   pequeña a escala cosmológica? (1 Mpc $= 3.09\times10^{22}$ m.)

</div>

In [7]:
s_umbral = (m_p + m_pi0)**2
E_gamma  = (s_umbral - m_p**2) / (2 * m_p)
E_p      = s_umbral / (4 * 1e-12)                 # fotón de 1 meV = 1e-12 GeV, choque frontal
lam      = 1 / (4.1e8 * 0.6e-31)                  # m  (0.6 mb = 0.6e-31 m^2)
print('E_gamma = {:.0f} MeV;  E_p = {:.1e} eV;  lambda = {:.1e} m = {:.1f} Mpc'.format(
      1e3 * E_gamma, 1e9 * E_p, lam, lam / 3.09e22))

E_gamma = 145 MeV;  E_p = 2.9e+20 eV;  lambda = 4.1e+22 m = 1.3 Mpc


<details>
<summary><b>Solución</b></summary>

En el umbral, $s = (m_p + m_\pi)^2$.

1. $s = (E_\gamma + m_p)^2 - E_\gamma^2 = m_p^2 + 2m_pE_\gamma \Rightarrow
   E_\gamma = \dfrac{(m_p + m_\pi)^2 - m_p^2}{2m_p} = 145$ MeV.
2. Con $E_p \simeq p_p$: $s \simeq 2E_\gamma E_p(1 - \cos\theta)$, mínimo en choque frontal ($\cos\theta = -1$):
   $s = 4E_\gamma E_p \Rightarrow E_p = \dfrac{(m_p + m_\pi)^2}{4E_\gamma} = 2.9\times10^{20}$ eV.
   Por encima, los protones pierden energía con el fondo cósmico (corte GZK).
3. $\lambda = 1/(\rho_\gamma\sigma) = 4\times10^{22}$ m $\simeq 1.3$ Mpc, frente a los $\sim10^{4}$ Mpc del
   universo observable: **minúscula**. Un protón de $10^{20}$ eV no puede venir de lejos.
   (Matiz honesto: 1.3 Mpc es la longitud de interacción *muy por encima* del umbral. La distancia que de
   verdad importa es la de pérdida de energía, más larga —del orden de 100 Mpc—, porque cerca del umbral solo
   participa la cola energética del espectro de cuerpo negro y porque cada colisión se lleva solo un
   $\sim20\,\%$ de la energía del protón. La conclusión no cambia: hay un horizonte GZK, y los rayos cósmicos
   de energía extrema tienen que proceder del universo cercano.)

</details>

## Fracción de desintegración y anchura

<div class="admonition tip">
<p class="admonition-title"><b>[>] Ejercicio 9 — Higgs a dos fotones en el LHC</b></p>

La sección eficaz de producción del Higgs es 50 pb en colisiones $p + p$ a $\sqrt s = 8$ TeV. Si
$\mathcal{B}(H \to \gamma + \gamma) = 2.2\times10^{-3}$, ¿cuántos sucesos de este tipo se producen en un año
($10^7$ s) con $\mathcal{L} = 10^{34}$ $\mathrm{cm}^{-2}\,\mathrm{s}^{-1}$? ¿Qué luminosidad integrada hace falta para
producir 100?

</div>

In [8]:
sigma, Br, lum, T = 50 * pb, 2.2e-3, 1e34, 1e7
print('N = {:.1e};  L_int para 100 = {:.0f} pb^-1'.format(sigma * Br * lum * T, 100 / (50 * Br)))

N = 1.1e+04;  L_int para 100 = 909 pb^-1


<details>
<summary><b>Solución</b></summary>

$N = \sigma\,\mathcal{B}\,\mathcal{L}\,T = 50\times10^{-36}\times2.2\times10^{-3}\times10^{34}\times10^{7} = 1.1\times10^{4}$.
$\mathcal{L}_{\rm int} = N/(\sigma\mathcal{B}) = 100/(50\ \mathrm{pb}\times2.2\times10^{-3}) = 909$ $\mathrm{pb}^{-1}$
$\simeq 0.9$ $\mathrm{fb}^{-1}$.

</details>

<div class="admonition tip">
<p class="admonition-title"><b>[>] Ejercicio 10 — Anchura y vida media</b></p>

La vida media del $\pi^0$ es $\tau = 8.4\times10^{-17}$ s. ¿Cuál es su anchura de desintegración? La anchura
del mesón $\eta$ es $\Gamma_\eta = 1.3$ keV. ¿Cuál es su vida media?

</div>

In [9]:
print('Gamma(pi0) = {:.1f} eV;  tau(eta) = {:.1e} s'.format(1e9 * hbar / 8.4e-17, hbar / 1.3e-6))

Gamma(pi0) = 7.8 eV;  tau(eta) = 5.1e-19 s


<details>
<summary><b>Solución</b></summary>

$\Gamma = \hbar/\tau$: $\Gamma_{\pi^0} = 6.58\times10^{-16}$ eV s $/ 8.4\times10^{-17}$ s $= 7.8$ eV;
$\tau_\eta = \hbar/\Gamma_\eta = 6.58\times10^{-16}/1.3\times10^{3} = 5.1\times10^{-19}$ s.
Anchuras de eV–keV para desintegraciones electromagnéticas; el $\eta$ vive menos porque tiene más canales
abiertos.

</details>

## Datos

**Constantes.** $\hbar c = 197.3$ MeV fm; $\hbar = 6.582\times10^{-22}$ MeV s; $c = 3.0\times10^{8}$ m/s;
$N_A = 6.022\times10^{23}$ $\mathrm{mol}^{-1}$; 1 b $= 10^{-24}$ $\mathrm{cm}^2 = 100$ $\mathrm{fm}^2$, 1 pb $= 10^{-36}$ $\mathrm{cm}^2$.

**Masas.** $m_{\pi^0} = 134.98$ MeV, $m_p = 938.27$ MeV, $m_\tau = 1776.9$ MeV, $m_Z = 91.19$ GeV.